Homework8_2/RealEstateObjectExceptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Homework8_2
+{
+    public class RealEstateObjectExceptions : Exception
```

Comment:
According to the task you class should contain additional variables. E.g. price and in public RealEstateObjectExceptions(int price) you can compare parameter price with your field and this will determine the text of error message